doc/examples/movie01/movie01.sh

```diff
@@ -18,6 +18,7 @@
 # https://www.youtube.com/watch?v=LTxlR5LuJ8g
 # The movie took ~6 hours to render on a 24-core MacPro 2013.
 
+gmt set PROJ_LENGTH_UNIT inch FONT_TAG 20p,Helvetica,white
```

Comment:
Move this line just before the "gmt movie" command?

Reply:
Yep done